pkgs/by-name/co/conda/package.nix

```
@@ -44,10 +44,23 @@
 # $ conda install spyder
 let
   version = "24.9.2";
-  src = fetchurl {
-    url = "https://repo.anaconda.com/miniconda/Miniconda3-py312_${version}-0-Linux-x86_64.sh";
-    hash = "sha256-jZNrpgAwDgjso9h03uiMYcbzkwNZeytmuu5Ur097QSI=";
-  };
+  selectSystem = attrs: attrs.${stdenv.hostPlatform.system};
```

```suggestion
  selectSystem = attrs: attrs.${stdenv.hostPlatform.system} or (throw "conda: ${stdenv.hostPlatform.system} is not supported");
```

Would be nice to have a fallback just incase someone uses `allowUnsupportedSystem` or similar